apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -569,6 +587,39 @@ emqx_schema {
     }
   }
 
+  persistent_session_builtin_session_table {
+    desc {
+         en: "Performance tuning options for built-in session table."
+         zh: "对内置的会话表进行调整。"
+    }
+    label: {
+         en: "Persistent session."
+         zh: "持久会话。"
+    }
+  }
+
+  persistent_session_builtin_sess_msg_table {
+    desc {
+         en: "Performance tuning options for built-in session messages table."
```

Comment:
I'm trying to remember: are the "session messages" pointers from (possibly) different persistent sessions to the message payload itself (stored in the "messages" table)?

Reply:
Yes.